### PR TITLE
Rebase off of create-react-app

### PR DESCRIPTION
**Pull request by suzil**
There was an issue with running the app locally (ReferenceError: exports is not defined). I had trouble pinpointing what caused the issue, so I decided to rebase off of CRA and update all packages. Now the local app runs without issue.

App seems to work as before with the changes:

### CODE REVIEW

File: frontend/src/radis.d.ts [hunk 1]
```diff
@@ -1,9 +0,0 @@
-// declare module "*.jpg" {
-//   const value: string;
-//   export = value;
-// }
-// declare module "*.png" {
-//   const value: string;
-//   export = value;
-// }
-declare module "*.png";
```
Comment (suzil) [inline frontend/src/radis.d.ts:9]: Kind of seems like we don't need this anymore 🤷‍♀️

File: frontend/package.json [hunk 2]
```diff
@@ -5,33 +5,27 @@
   "dependencies": {
     "@emotion/react": "^11.9.3",
     "@emotion/styled": "^11.9.3",
-    "@hookform/resolvers": "^2.9.3",
-    "@material-ui/core": "^4.11.3",
-    "@material-ui/icons": "^4.11.3",
-    "@material-ui/lab": "^4.0.0-alpha.57",
```
Comment (suzil) [inline frontend/package.json:11]: We use the MUI package now so we don't need these.

File: frontend/.env [hunk 3]
```diff
@@ -1 +0,0 @@
-SKIP_PREFLIGHT_CHECK=true
```
Comment (suzil) [inline frontend/.env:1]: Seems like we don't need this?

Comment (arunavabasu-03) [reply frontend/.env:1]: yup !! @suzil

**Comment by arunavabasu-03** [comment]
looks good @suzil !! 😀

**Pull request merged**